pkg/volumeclaimbinder/persistent_volume_claim_binder.go

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volumeclaimbinder
```

Comment:
This binder (and the proposed PersistentVolumeControllerManager for provisioning) are intended to be controller plugins.  I think packaging them beneath /controllers/{plugin-name} like Volumes makes sense.